src/agent/onefuzz-file-format/src/coverage/source.rs

```diff
@@ -8,15 +8,29 @@ pub mod v0;
 pub mod v1;
 
 #[derive(Serialize, Deserialize)]
-#[serde(tag = "version")]
+#[serde(tag = "version", content = "coverage")]
 pub enum SourceCoverageJson {
-    #[serde(rename = "0")]
+    #[serde(rename = "0.1")]
     V0(v0::SourceCoverageJson),
 
-    #[serde(rename = "1")]
+    #[serde(rename = "1.0")]
     V1(v1::SourceCoverageJson),
 }
 
+impl SourceCoverageJson {
+    pub fn deserialize(text: &str) -> Result<Self> {
```

Comment:
This is what we will use to load old and new files for further processing.

We will round-trip from old versions to new via the fallible conversions to the appropriate non-`Json` coverage type.